r/aoa_script.R

```diff
@@ -16,6 +16,7 @@ library(rstac) #rstac for accessing STAC-Catalogue
 library(rjson) #rjson for reading json input job file
 library(raster) #raster-Package for working with various raster formats
 library(gdalcubes) #gdalcubes-Package for creating, handling and using spatio-temporal datacubes
+library(kernlab) #kernlab for training kernel based support vector machines
```

Comment:
Müssen wir dies in Docker angeben? Hast du dies schon mal ausprobiert?

Reply:
Denk schon das dass auf dem Container installiert werden muss. Habs aber noch nicht getestet...